build.gradle

```diff
@@ -55,6 +55,8 @@ subprojects {
 
     dependencies {
 
+        implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: project.ext.awsLambdaCore
```

Comment:
This library is used by all lambdas so it makes sense to be in the common dependencies.